### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by storres20**
# In this milestone I did the following activities:
- I add "testing" branch
- I add react-test-renderer package
- I add testing files for components, pages
- I add testing file for calculate.js and operate.js
- I add snapshots files
- I checked the linters

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @storres20,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### This is a group project. Kindly ensure your two other coding partners contribute to this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by storres20** [comment]
Thanks a lot!!! I forgot about this is a group project

**Review by TadesseDev — COMMENTED** [review]
### Hi @storres20 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

- [ ] Dear @storres20, This is a pair programing exercise and we are supposed to see both of the collaborators make a contribution before reviewing the project. In your case I can only see your commit, This is an invalid review unless there exists a contribution by all the collaborators.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/tests/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  });
+
+  test('Multy operation', () => {
+    expect(operate(9, 9, 'x')).toBe('81');
+  });
+
+  test('Modulo operation', () => {
+    expect(operate(5, 4, '%')).toBe('1');
+  });
+});
```
Comment (zieeco) [inline src/tests/operate.test.js]: - Hey Team, good job with the test, however, some tests are failing, please, kindly check the Minus test to fix the issue. see the image below:

Screenshot from 2022-12-08 17-16-15


Screenshot from 2022-12-08 17-14-24

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to have your project approved, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions
- [ ] [OPTIONAL] Good job so far! Please remember that all coding partners must participate in `group projects`, but I saw that `Ric Martinez Aguilar` did not participate in yours. I approved this time because the code reviewer before me did not notice it. Please pay attention to it next time. If any of the coding partners refuse to work or do not participate, you should contact `Microverse student support` about this 🙏🏼 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**